graf2d/asimage/src/libAfterImage/ungif.c

```diff
@@ -172,7 +172,7 @@ get_gif_image_desc( GifFileType *gif, SavedImage *im )
 #else
 			im->ImageDesc.ColorMap = MakeMapObject(gif->Image.ColorMap->ColorCount, NULL);
 #endif
-			fseek( gif->UserData, start_pos+9, SEEK_SET ); 
+			fseek( gif->UserData, start_pos+9, SEEK_SET );
```

Comment:
May I ask you to revert this hunk? I'd like to be able to fetch this patch and apply it to the upstream tarball.